package/ntrip_daemon/src/ntrip_daemon.c

```diff
@@ -291,7 +276,9 @@ static int ntrip_settings_loop(void)
                        ntrip_settings_init,
                        ntrip_reconnect,
                        settings_loop_terminate,
-                       settings_loop_sigchild) ? 0 : -1;
+                       settings_loop_sigchild) ?
```

Comment:
Wtf?

Reply:
I can live with this, just looks pretty weird with this particular chunk of code.